content/blog/2020/2020-04-02-covid-19-basic.md

```diff
@@ -0,0 +1,205 @@
+
+The message here regards the essential difference between plotting curves and real data mining: static curves are dead; data mining is **interactive**.
+
+## Connecting data sets
+
+Let's solve the problem of negligible curves for small countries. This is not just about populations: the biggest problem is that confirmed cases are the result of testing, and testing strategies vary by country. Icelanders and Koreans test like crazy, while the number of tests in the US was (initially) so small it was comparable to that of Slovenia (and with it, the number of confirmed cases). The number of tests could thus **normalize** the number of confirmed cases, but this wouldn't do either; the testing isn't equivalent – some countries test more at random, while others save tests for groups at greater risk. And regardless, exact data about the number of tests is not readily available.
+
+So, let's fallback to the number of cases per million inhabitants. Neither this nor the population of countries are present in the data provided by John Hopkins University. We do, however, have another data set at arm's reach: the population of countries (for year 2015) appears in the Worldbank's Human development index (HDI) data. To load it, use the Datasets widget, find and double-click the HDI data set.
+
+Now we're working with two data sets. Let's merge them using the Merge Data widget: connect both the File widget (with the John Hopkins data), and the Datasets widget (with the HDI data) to Merge Data. Make sure to match the "Country/Region" feature in COVID-19 data with the "Country" feature in HDI. It is also important to connect the widgets in this order, so File provides the main data and the Datasets widget augments it with additional annotation columns. If you do it incorrectly, double-click the connections to fix it.
+
+{{% figure src="/blog_img/2020/2020-04-02-merge-data.png" %}}
+
+The match is imperfect: some countries appear with different names, for instance, `Russia` from John Hopkins doesn't match the `Russian Federation` from HDI, and the same goes for John Hopkin's `US` and HDI's `United States`. 
+
+The population of Liechtenstein in millions with one decimal is 0.0; similar for Palau. To remove countries with unknown or zero population, we continue with Select Rows, where we set the condition "Total Population (millions) 2015 is greater than 0".
+
+Now for some constructive business. To get the number of cases on some particular day, we use the Feature Constructor widget, where we can input a formula to compute new data columns. 
+In "New" we select Numeric, 
```

Comment:
This spacing doesn't work in the actual blog. You need to use backslash for that.